### DISCUSSION TITLE
Using existing zookeeper

### DISCUSSION THREAD

**Discussion by krojew** [strimzi/strimzi-kafka-operator#5588] [Q&A] [answered]
Hi, I'm interested in using strimzi, but there's one blocker I can't find any information on how to approach. I already have a deployed zookeeper cluster, so I don't want to have another one deployed by strimzi, but simply use the existing one. Is there a way to configure such use case?

**Comment by scholzj** [comment, marked as answer]
No, this is not supported in Strimzi, sorry.

**Reply by krojew** [reply]
Ah, too bad. Are there any future plans for such use case, maybe?

**Reply by scholzj** [reply]
No, the focus as with Apache Kafka itself is on removing ZooKeeper completely.
